components/brave_extension/extension/brave_extension/youtubedown.js

```diff
@@ -0,0 +1,3053 @@
+/* Copyright © 2019-2020 Jamie Zawinski <jwz@jwz.org>
```

Comment:
how is this going to be maintained? It seems like it should be a downloadable component?

Reply:
it also needs some kind of directory structure so we can modularize playlist inside of the brave extension